### ISSUE TITLE
Incorrect history data

### ISSUE THREAD

**Issue by SShikhali** [GPUOpen-Effects/FidelityFX-SSSR#4]
Hi! 
When debugging with RenderDoc, I noticed that some buffers are not cleared properly (camera movements are not handled correctly).

![image](https://github.com/GPUOpen-Effects/FidelityFX-SSSR/assets/99796443/228cd62e-e412-4b25-9ab3-87c8012e0dc0)

**Comment by frguthmann** [comment]
Hi,

Thanks for looking at SSSR! This resource is not being fully cleared by design. The denoiser works on tiles and will only overwrite the tiles it is working on. The rest of the texture remains untouched.

It is of course always possible that we have a bug ! If you have noticed any visual artifact coming from the lack of clearing I'm happy to investigate.

Please note that this specific version of SSSR is not maintained anymore. SSSR has been integrated in our SDK where it will receive all future updates and bug fixes.
https://github.com/GPUOpen-LibrariesAndSDKs/FidelityFX-SDK/blob/main/docs/techniques/stochastic-screen-space-reflections.md

**Comment by SShikhali** [comment]
Yeah, I thought that was the reason, and I guess that's not an issue if the algorithm uses an old data from the history for one frame. Thanks for the reply and the link to a new version!

**Comment by SShikhali** [comment]
Hi, @frguthmann. Sorry to bother, I didn't want to open another issue.

Just wanted to let you know that there're some artifacts in SSSR. And it's reproducible both on this version, and a newer SDK version. When moving camera around, some blue spots may appear and it looks like the issue comes from the radiance history.
FYI, I have RX 7900XTX.

![image](https://github.com/GPUOpen-Effects/FidelityFX-SSSR/assets/99796443/7097126d-5a76-4bec-a8fa-57fef546d0c9)
![image](https://github.com/GPUOpen-Effects/FidelityFX-SSSR/assets/99796443/3fae876f-6193-4dc2-a60c-96878de4a9b9)

**Comment by frguthmann** [comment]
No problem, thanks for reporting this. Unfortunately I can't reproduce on my 7900XTX. I can see you're running the Vulkan version of the sample, did you change any of the settings? Can you identify anything specific to your setup that would trigger this?

I tried moving the camera slowly and pretty fast but nothing. Anything that would help me reproduce is appreciated :).

**Comment by SShikhali** [comment]
No, I didn't change anything. DX runs fine though.

GPU: RX 7900XTX, driver: 24.5.1 (latest at the moment)
Vulkan SDK: 1.3.275
Visual Studio: 17.9.6
Let me know if there's anything else that I should provide to you.

https://github.com/GPUOpen-Effects/FidelityFX-SSSR/assets/99796443/2ba2952a-3a6c-4db8-8f80-ffba579cce67

**Comment by frguthmann** [comment]
Ah you're right, I wasn't testing the correct version! It looks like the latest available version on `main` in the FidelityFX-SDK repository does exhibit this issue with Vulkan. I have checked and the branches/packages released for FSR3 fixed this issue already. ( Despite the sample looking desaturated )

The better news is I have checked the latest internally available version of the SDK, the issue is not present and the sample is looking good. I can't give you any ETA but the next update of the SDK will fix it :)
